libr/debug/p/native/xnu/xnu_debug.c

```diff
@@ -516,53 +516,12 @@ int xnu_get_vmmap_entries_for_pid (pid_t pid) {
 	return n;
 }
 
-static int get_bits () {
```

Comment:
The compiler said it wasn't used. The only place where it's used is SAME_BITNESS but that macro is not used anywhere.